contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -201,7 +202,33 @@ pltsql_createFunction(ParseState *pstate, PlannedStmt *pstmt, const char *queryS
 		return false;
 	}
 	else
-	{	
+	{
+		/* Do not check for duplicate function if REPLACE flag is set and during dump and restore. */
```

Comment:
What if we do a replace with same name but different number of arguments? will that fail?

Reply:
That can still be done via PG port. 
`replace` flag will never be set to true for T-SQL since we don't support `CREATE [OR ALTER] function` in T-SQL.